app/github_repo_fetcher.rb

```diff
@@ -11,6 +11,8 @@ def repo(app_name)
   # Fetch a README for an alphagov application and cache it.
   # Note that it is cached as pure markdown and requires further processing.
   def readme(app_name)
```

Comment:
Should this class use `github_repo_name` instead of `app_name`, for clarity?

Reply:
Good idea - I've tacked on a commit.